scripts/amd/gemm/tune_gemm.py

```diff
@@ -673,7 +690,7 @@ def main():
         print(f"Tuning starts at: {start_time}", flush=True)
         f_results = open(tuning_output_file, 'w')
 
-    for (M, N, K, col_a, col_b, myConfig) in mnks:
+    for (M, N, K, col_a, col_b, init_type, myConfig) in mnks:
```

Comment:
I think we can remove `init_type` from `mnks`. All gemm sizes in a single tuning/benchmarking session can shared the same `init_type`